### ISSUE TITLE
LVM package not installed by default

### ISSUE THREAD

**Issue by thomasfinstad** [steamos-community/stephensons-rocket#114]
I just installed the newest version `StephensonsRocket2.49plus1.iso`, I used expert mode so I could add my extra HDD to the `/home/` partition with LVM.

When I booted up it was stuck at the grub screen, I changed to TTY2 and was able to see that systemd tried to start a job for the lvm volume I created, when it failed to mount I got a terminal and found the `lvm2` package to be missing, installed that package and rebooted, seems to work fine after that.

Only things I changed from default install in expert mode was location and the `/home` partition to be an lvm volume consisting of 2 disks. Machine is UEFI if that matters during package selection the installer does.

**Comment by directhex** [comment]
Interesting. I'll try to investigate when I get the chance.

**Comment by grafov** [comment]
I got same problem. I used expert installation and created 2 LVM groups (one on SDD and another on HDD). Then I created /home on LVM. Install process completed ok but after reboot I got maintanance mode with not mounted /home. `lvm` binary was missed. I manually configured network then made `apg-get update; apt-get install lvm`. It solved problem. My machine uses BIOS boot not UEFI.

**Comment by sharkwouter** [comment]
I can confirm this issue. I just did a custom install with an lvm root partition and my initramfs didn't even contain lvm.

**Comment by sharkwouter** [comment]
I think I found the problem. When forcing the installer to install the lvm2 package, the installer can not finish because it fails to install some packages. Here is a picture of the log:
![stephensonsrocket1](https://cloud.githubusercontent.com/assets/5042659/12175473/c69f72be-b562-11e5-99ad-d7b02dd04d0d.png)

As seen [here](https://packages.debian.org/jessie/lvm2), those are dependencies for for the lvm2 package.

**Comment by sharkwouter** [comment]
The following lvm2 dependencies were missing from the disk:

dmeventd
libreadline5
liblvm2cmd2.02

I added these packages from the SteamOS repo, build and iso and did a test install with lvm and it worked.

I have pushed the packages to the Github repo. Lvm will work if you build your own iso or wait for the next release.

edit: this is the commit https://github.com/steamos-community/stephensons-rocket/commit/a95c691649e8abd7a805ba028a8797b4ff227ac6
